docs/src/api/params.md

```diff
@@ -1135,6 +1140,7 @@ When set to `"hide"`, screenshot will hide text caret. When set to `"initial"`,
 - %%-screenshot-option-caret-%%
 - %%-screenshot-option-type-%%
 - %%-screenshot-option-mask-%%
+- %%-screenshot-option-mask-color-%%
```

Comment:
Since this option was not added in v1.8, you'll have to manually add it everywhere `screenshot-options-common-list-v1.8` is mentioned, remove it from this list, and insert `since: v1.34` before line 1096.